src/main/java/CoinLocker/GetUserInput.java

```diff
@@ -0,0 +1,7 @@
+package CoinLocker;
+
+public interface GetUserInput {
```

Comment:
For real project, we should add javadoc and comments for public interface and methods.

Reply:
Javadoc added